src/providers/details/ProviderDetailsPage.tsx

```diff
@@ -0,0 +1,69 @@
+import type { FC } from 'react';
+import { LoadingDots } from 'src/components/common/LoadingDots/LoadingDots';
+import { ErrorState } from 'src/components/common/Page/PageStates';
+import { useForkliftTranslation } from 'src/utils/i18n';
+
+import { ProviderModelGroupVersionKind, type V1beta1Provider } from '@kubev2v/types';
+import { type K8sModel, useK8sWatchResource } from '@openshift-console/dynamic-plugin-sdk';
+
+import OpenshiftProviderDetailsPage from './OpenshiftProviderDetailsPage';
+import OpenStackProviderDetailsPage from './OpenStackProviderDetailsPage';
+import OvaProviderDetailsPage from './OvaProviderDetailsPage';
+import OVirtProviderDetailsPage from './OVirtProviderDetailsPage';
+import VSphereProviderDetailsPage from './VSphereProviderDetailsPage';
+
+import './ProviderDetailsPage.style.css';
+
+type ProviderDetailsPageProps = {
+  kind: string;
+  kindObj: K8sModel;
+  name: string;
+  namespace?: string;
+};
+
+const LoadError = () => {
```

Comment:
export to a new component

Reply:
No need for this component, I canceled it.